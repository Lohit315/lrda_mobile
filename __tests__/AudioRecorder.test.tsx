import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import React from 'react';
import { shallow } from 'enzyme';
import AudioContainer from '../lib/components/audio.tsx';

Enzyme.configure({ adapter: new Adapter() });

describe('AudioContainer', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<AudioContainer newAudio={[]} setNewAudio={() => {}} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('simulates clicking record and successfully uploading audio', () => {
    const wrapper = shallow(<AudioContainer newAudio={[]} setNewAudio={() => {}} />);
    wrapper.find({ testID: 'startRecordingButton' }).simulate('press');

  });
});