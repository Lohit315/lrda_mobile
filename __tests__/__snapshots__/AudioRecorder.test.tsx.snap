// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioContainer renders without crashing 1`] = `ShallowWrapper {}`;
